olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h

```diff
@@ -31,25 +31,59 @@ namespace dataservice {
 namespace read {
 
 /**
- * @brief Represents one stream layer consumer config entry.
+ * @brief Represents a consumer configuration entry of one stream layer.
  *
- * An entry can be one of: string, int32, bool.
+ * Examples: `string`, `int32`, `bool`.
```

Comment:
The previous comment denoted that the accepted variable types are string, int32 or bool.
Your comment implies you should use these values as string input. Or are you also referring to variable data types?

Reply:
Changed